requirements.txt

```diff
@@ -1,35 +1,35 @@
-APScheduler==3.6.1
+APScheduler==3.6.3
 autobahn==20.7.1
-beautifulsoup4==4.8.0
-colorama==0.4.1
+beautifulsoup4==4.9.1
+colorama==0.4.3
 cssmin==0.2.0
 Flask==1.1.2
 Flask-Assets==2.0
-Flask-RESTful==0.3.7
+Flask-RESTful==0.3.8
 Flask-Scrypt==0.1.3.6
 httplib2==0.18.1
-irc==17.1
+irc==19.0.0
 isodate==0.6.0
 jsmin==2.2.2
-Markdown==3.1.1
-Pillow==7.1.0
+Markdown==3.2.2
+Pillow==7.2.0
 psycopg2==2.8.5
-pylast==3.1.0
+pylast==3.3.0
 pyOpenSSL==19.1.0 # for twisted.internet.ssl
 pyScss==1.3.7
 python-Levenshtein==0.12.0
-pytz==2019.2
+pytz==2020.1
 ratelimiter==1.2.0.post0
-redis==3.3.8
-regex==2019.8.19
```

Comment:
no changelog I could find. also as you can see, no semantic versioning. however this exists: https://bitbucket.org/mrabarnett/mrab-regex/commits/

and it only looks like minor changes/bugfixes.

✅